src/systems/ec/specs/components/card/docs/code.mdx

```diff
@@ -0,0 +1,17 @@
+import Playground from '@ecl/website-components/Playground'
+import Card from '@ecl/ec-react-component-card/Card'
+
+import demoContentCard from '../demo/data--card';
+import demoContentTile from '../demo/data--tile';
+
+## Card
+
+<Playground playgroundLink="/storybook/ec/index.html?selectedKind=Card&selectedStory=card&stories=0">
```

Comment:
You can use the new syntax here (check https://github.com/ec-europa/europa-component-library/blob/next-v2/src/systems/ec/specs/components/hero-banner/docs/code.mdx#L13)